extensions/amp-video/1.0/video-wrapper.js

```diff
@@ -269,8 +269,14 @@ function Autoplay({
       {displayOverlay && (
         <div
           role="button"
+          tabindex="0"
           style={fillContentOverlay}
           onClick={onOverlayClick}
+          onKeyPress={(e) => {
+            if (e.which === /* enter */ 13) {
+              e.target.click();
+            }
+          }}
```

Comment:
Trying to prevent drifting on Bento. I wonder if we need shared a11y tests for both 0.1 / 1.0.